src/resources/i18n/translations/en.json

```diff
@@ -510,8 +511,8 @@
   "QR Code Error": "QR Code Error:",
   "QR Reader": "QR Reader",
   "Queue": "Queue",
-  "Queue Last": "Queue: Last",
-  "Queue Next": "Queue: Next",
+  "Queue Last": "Queue: last",
+  "Queue Next": "Queue: next",
```

Comment:
These are for buttons that appear in an alert. I think of alert labels as generally all caps except prepositions and articles. The lowercase looks a little dull to me for some reason...I'll try it to see how it looks, but I think button labels are better-off uppercase? They're not really list items so much as buttons in a menu.

Reply:
And En-queue first/last, or (better I think) play last/first is off the table?